controller/kubernetes_pod_controller.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	controllerAgentName = "longhorn-kubernetes-pod-controller"
+
+	remountDelayDuration = 5
```

Comment:
```go
remountDelayDuration = 5 * time.Second
```

Reply:
The time will be handled here
`delayDuration := time.Duration(int64(remountDelayDuration)) * time.Second`

time.Duration accepts a int64